docs/reference/config-example.md

```diff
@@ -25,12 +25,32 @@ services:                                                # compose services sect
       kev.workload.max-cpu: "0.2"                        # Default: 0.2. CPU limit per workload.
       kev.workload.memory: 10Mi                          # Default: 10Mi. Memory request per workload.
       kev.workload.max-memory: 500Mi                     # Default: 500Mi. Memory limit per workload.
-      kev.workload.liveness-probe-disabled: "false"      # Default: false. Disable/Enable liveness probe.
-      kev.workload.liveness-probe-command: "echo 'n/a'"  # Liveness probe command to run.
-      kev.workload.liveness-probe-initial-delay: 1m0s    # Default: 1m. How long to wait before initial probe run.
-      kev.workload.liveness-probe-interval: 1m0s         # Default: 1m. Interval for the probe.
-      kev.workload.liveness-probe-retries: "3"           # Default: 3. Number of probe retires.
-      kev.workload.liveness-probe-timeout: 10s           # Default: 10s. Probe command timeout.
+      kev.workload.liveness-probe-type: "command"        # Default: command. Possible options: none | command | http | tcp.
+      ## Liveness probe
+      ### Command
+      kev.workload.liveness-probe-command: "echo 'n/a'"  # Liveness probe command to run. Only used when using command probe type.
+      ### HTTP
+      kev.workload.liveness-probe-http-port: 8080        # HTTP Liveness probe port. Only used when using an http probe type.
+      kev.workload.liveness-probe-http-path: /status     # HTTP Liveness probe path. Only used when using an http probe type.
+      ### TCP
+      kev.workload.liveness-probe-tcp-port: 8080         # TCP Liveness probe port. Only used when using an tcp probe type.
+      kev.workload.liveness-probe-initial-delay: 1m0s    # Default: 1m. How long to wait before initial probe run. Shared by all liveness probe types.
```

Comment:
perhaps move the last four lines up to the liveness probe as they are common / shared across all liveness probe types. Similar comment for the readiness probe below.

Reply:
but aren't these secondary? as in, they even have defaults and can be omitted. What about adding a ### Common section?

Reply:
yes please. commend would do.. something to tell that these are common for all liveness probe types.